apps/sms/test/marionette/conversation_test.js

```diff
@@ -491,4 +503,83 @@ marionette('Conversation Panel Tests', function() {
       });
     });
   });
+
+  suite('Add recipients from thread', function() {
+    var conversationView;
+    setup(function() {
+      var thread = ThreadGenerator.generate({
+        numberOfMessages: 1
+      });
+      messagesApp.launch();
+
+      storage.setMessagesStorage([thread], ThreadGenerator.uniqueMessageId);
+      // Set empty contacts store.
+      storage.setContactsStorage();
+      var inboxView = new InboxView(client);
+      conversationView = inboxView.goToConversation(thread.id);
+      messagesApp.Composer.messageInput.sendKeys('message2');
+    });
```

Comment:
mmm I don't know it's failing here but not elsewhere.

That said, since bug 1200583 we changed how the storage work:
* you don't need to specify empty storages (so you can remove the contact storage)
* you need to set them _before_ launch.

maybe this will automagically fix the issue :)